Source/WebCore/animation/KeyframeEffect.cpp

```diff
@@ -1597,6 +1599,20 @@ void KeyframeEffect::setAnimatedPropertiesInStyle(RenderStyle& targetStyle, doub
                     if (endKeyframeCompositeOperation != CompositeOperation::Replace)
                         CSSPropertyAnimation::blendProperties(this, cssPropertyId, endKeyframeStyle, targetStyle, *endKeyframe.style(), 1, endKeyframeCompositeOperation);
                 }
+
+                // If this keyframe effect has an iteration composite operation of accumulate,
+                if (m_iterationCompositeOperation == IterationCompositeOperation::Accumulate) {
+                    // apply the following step current iteration times:
```

Comment:
And this comment isn't particularly helpful either.

Reply:
Yes, from the spec, like the rest of this method.